contracts/Cargo.toml

```diff
@@ -28,6 +28,7 @@ rand.workspace = true
 # features, because this crate is meant to be used in a `no_std` environment.
 # Currently, the std feature is only used for testing purposes.
 std = []
+export-abi = ["stylus-sdk/export-abi"]
```

Comment:
By declaring `export-abi` feature without activating it, we remove those pesky warnings. 
We also enable generating ABI if we deem it necessary, although this isn't as important atm.

Reply:
This causes other CI flows to fail, will investigate further

Reply:
The errors appear for some weird reason:

```
the trait `SolError` is not implemented for `[OUR_ERROR_NAME]`
```

It seems this happens when `--feature export-abi` is enabled in our cargo hack workflow.

Unless any of you guys have an idea how to mitigate this, I'd say we will have to live with export-abi warnings for now :confused: